lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb

```diff
@@ -17,7 +17,7 @@ class ConfigManager
     ).void }
     def initialize(sdkKey, local_bucketing, wait_for_init)
       @first_load = true
-      @config_version = "v1"
+      @config_version = "v2"
```

Comment:
do we use this anywhere? You'll also need to update the WASM at the same time as this config

Reply:
yeah it's used to fetch different versions of the config below